doc/docca/include/docca/doxygen.xsl

```diff
@@ -1209,7 +1209,7 @@
       $private &gt; 0">
     <xsl:text>[heading Private Data Members]&#xd;</xsl:text>
     <xsl:text>[table [[Name][Description]]&#xd;</xsl:text>
-    <xsl:for-each select="sectiondef[@kind='private-attrib' or @kind='private-static-attrib']/memberdef" mode="class-table">
+    <xsl:for-each select="sectiondef[@kind='private-attrib' or @kind='private-static-attrib']/memberdef"  >
```

Comment:
There seems to be extra spacing before the `>` ?

Reply:
Removed.